util/hostnamer_test.go

```diff
@@ -0,0 +1,40 @@
+package util
+
+import (
+	"testing"
+	"time"
+)
+
+func TestHostNamerQuad8(t *testing.T) {
```

Comment:
Also include the IPv6 version for https://developers.google.com/speed/public-dns/docs/using#google_public_dns_ip_addresses

Reply:
OK.